x/fee_grant/client/cli/service_msg_client.go

```diff
@@ -0,0 +1,43 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	gogogrpc "github.com/gogo/protobuf/grpc"
+	grpc "google.golang.org/grpc"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var _ gogogrpc.ClientConn = &serviceMsgClientConn{}
+
+// serviceMsgClientConn is an instance of grpc.ClientConn that is used to test building
+// transactions with MsgClient's. It is intended to be replaced by the work in
+// https://github.com/cosmos/cosmos-sdk/issues/7541 when that is ready.
+type serviceMsgClientConn struct {
+	msgs []sdk.Msg
+}
```

Comment:
this is a interceptor mock - do we need it here? Is there other _ready_ Interceptor we can use?